client/src/components/DatasetInformation/DatasetAttributes.vue

```diff
@@ -26,7 +26,8 @@ const props = defineProps<Props>();
 const historyStore = useHistoryStore();
 
 const loading = ref(false);
-const messageText = ref("");
+const loadingFailed = ref(false);
+const messageText = ref<string | null>(null);
```

Comment:
> My thinking is that we need to separate the two error conditions, if submitting the value failed we shouldn't disable the rest of the interface

Yeah! That looks way better! Thank you!

Minor thing, undefined can serve here as well no need for extra null handling.
```suggestion
const messageText = ref<string>();
```

Reply:
don't we need to include undefined then in the type hint ?

Reply:
It should be inferred by leaving the value empty

![image](https://github.com/galaxyproject/galaxy/assets/46503462/00537089-599b-4a24-8972-676bfe61a979)